optimade/filtertransformers/mongo.py

```diff
@@ -505,3 +503,5 @@ def replacement(d, prop, expr):
         return _cached_filter
 
     return filter_
+
+    return filter_
```

Comment:
I have deleted this about 10 times locally, but rebasing has always reintroduced it...

```suggestion
```